### PR TITLE
Mhilton mychanges

### PR DESCRIPTION
**Pull request by marthaisabelhilton**
Small changes to lessons - some updates and wording/ grammar.

### CODE REVIEW

**Comment by marthaisabelhilton** [comment]
The checks are failing with:
`Error: Not found : gitbook-plugin-katex-auto-render
The command "gitbook install" failed and exited with 1 during .`

File: first-analysis-steps/introduction-to-course.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ standalone reference when you need to revisit a particular topic.
 We'll be looking at:
 
 * How data flows through the LHCb processing chain, what software is involved, 
-  and how the data are stored;
+  and how the data is stored;
```
Comment (chrisburr) [inline first-analysis-steps/introduction-to-course.md]: Controversial but I approve 😆

**Comment by marthaisabelhilton** [comment]
Hi @chrisburr this still didn't work I'm not sure why...

File: first-analysis-steps/davinci.md [hunk 2]
```diff
@@ -118,22 +116,24 @@ above) and a set for everything else. Generally, you will want the latest
...
 
-These lessons use [DaVinci v44r6][v44r6], which was the latest Run 1/2 version at the
+These lessons use [DaVinci v45r1][v45r1], which was the latest Run 1/2 version at the
 time the text was last revised.
 
 [releases]: http://lhcbdoc.web.cern.ch/lhcbdoc/davinci/releases/
-[v44r6]: http://lhcbdoc.web.cern.ch/lhcbdoc/davinci/releases/v44r6/
+[v45r1]: http://lhcbdoc.web.cern.ch/lhcbdoc/davinci/releases/v45r1/
+
+**Note:** Older versions of DaVinci may not be available on the default platform ```x86_64-centos7-gcc8-opt```. To get around this we can pick the best suitable platform by using ```lb-run -c best DaVinci/vXXrYpZ ...```.
```
Comment (vlisovsk) [inline first-analysis-steps/davinci.md]: Should we somewhere define what do we call a "platform", and what the name of the platform means?

File: first-analysis-steps/loki-functors.md [hunk 3]
```diff
@@ -38,9 +38,16 @@ if ever.
 
 To understand what we can do with LoKi functors, we will pick up from where we 
 left off [exploring a DST interactively](interactive-dst.html).
-Open the DST and get the first candidate in the `D2hhPromptDst2D2KKLine` line:
+First open the DST as we did previously:
+
+```bash
+$ lb-run DaVinci/v44r6 ipython -i first.py 00070793_00000001_7.AllStreams.dst
+```
```
Comment (vlisovsk) [inline first-analysis-steps/loki-functors.md]: Should the DaVinci version be updated here?

File: first-analysis-steps/davinci.md (hunk 2 above)
Comment (chrisburr) [reply first-analysis-steps/davinci.md]: That's a good idea maybe with a link to [`HSF-TN-2018-01`](https://hepsoftwarefoundation.org/notes/HSF-TN-2018-01.pdf).

File: first-analysis-steps/bookkeeping.md [hunk 4]
```diff
@@ -85,8 +85,7 @@ running our DaVinci job. Click it, select “Save as a
 python file”, and add `.py` to the end of the text in “Save As...”. Clicking “Save” once again in the pop-up menu will start the
 download. Save this file somewhere you can find it again.
 
-A copy of the file we just downloaded is [available
-here](data/MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09c_Trig0x6138160F_Reco16_Turbo03_Stripping28r1NoPrescalingFlagged_ALLSTREAMS.DST.py).
+A copy of the file we just downloaded is available <a href="data/MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09c_Trig0x6138160F_Reco16_Turbo03_Stripping28r1NoPrescalingFlagged_ALLSTREAMS.DST.py" download target="_blank">here</a>.
```
Comment (vlisovsk) [inline first-analysis-steps/bookkeeping.md]: Might the CI failure be related to this change?

File: first-analysis-steps/loki-functors.md (hunk 3 above)
Comment (marthaisabelhilton) [reply first-analysis-steps/loki-functors.md]: Done thanks.

File: first-analysis-steps/bookkeeping.md (hunk 4 above)
Comment (marthaisabelhilton) [reply first-analysis-steps/bookkeeping.md]: There was a problem with the link I've just corrected this. However it is still failing so I don't think this caused it...

**Comment by chrisburr** [comment]
Thanks again @marthaisabelhilton!

**Pull request merged**